src/hotspot/share/gc/z/zMark.hpp

```diff
@@ -26,19 +26,17 @@
 
 #include "gc/z/zMarkStack.hpp"
 #include "gc/z/zMarkStackAllocator.hpp"
+#include "gc/z/zMarkStackEntry.hpp"
 #include "gc/z/zMarkTerminate.hpp"
 #include "oops/oopsHierarchy.hpp"
-#include "utilities/globalDefinitions.hpp"
```

Comment:
This file uses definitions brought by globalDefinitions.hpp, so I don't see the reason for removing it.